gcloud/pubsub/_gax.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""GAX wrapper for Pubsub API requests."""
+try:
+    # pylint: disable=no-name-in-module
+    from google.gax import CallOptions
+    from google.gax.errors import GaxError
+    from google.gax.grpc import exc_to_code
+    # pylint: enable=no-name-in-module
+except ImportError:  # pragma: NO COVER
+    _HAVE_GAX = False
+else:
+    _HAVE_GAX = True
+
+    from google.pubsub.v1.pubsub_pb2 import PubsubMessage
```

Comment:
Couldn't a user have https://pypi.python.org/pypi/google-gax but not the https://pypi.python.org/pypi/gax-google-pubsub-v1 extra? This would make this import fail in our "success" state.